ts/Core/Time.ts

```diff
@@ -347,7 +347,7 @@ class Time {
             if (
                 unit === 'Milliseconds' ||
                 unit === 'Seconds' ||
-                unit === 'Minutes'
+                (unit === 'Minutes' && this.getTimezoneOffset(date) % 3600000 === 0)
```

Comment:
Please add a reference to the issue number - we've been struggling with this before